ldhooks/evaluation_series_context.go

```diff
@@ -0,0 +1,46 @@
+package ldhooks
+
+import (
+	"github.com/launchdarkly/go-sdk-common/v3/ldcontext"
+	"github.com/launchdarkly/go-sdk-common/v3/ldvalue"
+)
+
+// EvaluationSeriesContext contains contextual information for the execution of stages in the evaluation series.
+type EvaluationSeriesContext struct {
```

Comment:
Whenever one of these value receivers is invoked, it'll need to copy I guess 2 pointers + sizeof(Context) + sizeof(Value).